File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/ChatCompletionChunkChoice.java
package com.sdkwork.backend.model;

public class ChatCompletionChunkChoice {
    private Integer index;
    private ChatCompletionChunkDelta delta;
    private LogProbInfo logprobs;
    private String finishReason;

    public Integer getIndex() {
        return this.index;
    }
    
    public void setIndex(Integer index) {
        this.index = index;
    }

    public ChatCompletionChunkDelta getDelta() {
        return this.delta;
    }
    
    public void setDelta(ChatCompletionChunkDelta delta) {
        this.delta = delta;
    }

    public LogProbInfo getLogprobs() {
        return this.logprobs;
    }
    
    public void setLogprobs(LogProbInfo logprobs) {
        this.logprobs = logprobs;
    }

    public String getFinishReason() {
        return this.finishReason;
    }
    
    public void setFinishReason(String finishReason) {
        this.finishReason = finishReason;
    }
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/chat_completion_chunk_choice.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ChatCompletionChunkChoice:
    """流式聊天完成响应中的选项对象，包含模型生成的响应内容块"""
    index: int
    delta: ChatCompletionChunkDelta = None
    logprobs: LogProbInfo = None
    finish_reason: str = None
